INSTALL

```diff
@@ -98,7 +98,7 @@ additional Cartopy functionality.
 **pyepsg** 0.2.0 or later (https://github.com/rhattersley/pyepsg)
     A simple Python interface to https://epsg.io
 
-**scipy** 0.10 or later (https://www.scipy.org/)
+**SciPy** 0.10 or later (https://www.scipy.org/)
     Python package for scientific computing.
 
```

Comment:
You could also argue that `six` should be capitalised since [the documentation ](https://pythonhosted.org/six/) sometimes capitalised the it to be `Six`